docs/user_guide/hopsworks/python.rst

```diff
@@ -8,11 +8,11 @@ Python Environment Basics
 
 Hopsworks provides a premade environment for machine learning and data science development using python 3.7. The environment contain the most popular machine learning libraries including TensorFlow, Keras, PyTorch and scikit-learn.
 
-The environment ensures compatibility between the machine learning libraries and CUDA/cuDNN in case of NVIDIA GPUs and ROCm in case of AMD GPUs.
+The environment ensures compatibility between the CUDA version and the installed TensorFlow and PyTorch versions for applications using NVIDIA GPUs.
 
-In each project there is an Anaconda environment, which is replicated at all data processing hosts in the cluster. This means that whenever an operation is issued through the UI such as, installing or uninstalling a library, it is replicated on all the Anaconda environments which keeps them in sync.
+For each project, an Anaconda environment can be managed, which is is stored in a Docker image. Issuing an operation from the Python UI such as installing or uninstalling a library also updates the Anaconda environment.
```

Comment:
There is one Anaconda environment for each project, and the environment is layered on top of the project's base Docker image. That is, a library installed in the project's Anaconda environment can be used in a Job or Jupyter notebook (Python or PySpark) run in the project. In the Python UI, libraries can be installed or uninstalled from the project's Anaconda environment.